jupyterhub/values.yaml

```diff
@@ -50,12 +50,13 @@ hub:
   imagePullPolicy: IfNotPresent
   pdb:
     enabled: true
+    minAvailable: 1
   networkPolicy:
     enabled: false
     egress:
       - to:
-          - ipBlock:
-              cidr: 0.0.0.0/0
+        - ipBlock:
+            cidr: 0.0.0.0/0
```

Comment:
# TL, DR:
I dislike list indentation matters! Do not spend time thinking on what I write, and try to forgive me for posting it even though I say it is a pointless read ^^ Just feels too bad not to click send after considering this for too long in the past and now.

---

I'm unhappy about options of indentation here, its like choosing between two bad options...

- The change you made is more readable.
- The one we had is the one that matches the linter and what we do throughout the project. If one got to choose and stick with one, this is the one to avoid some issues such as needing to consider what is provided, a list or a dict when using the toYaml function and indenting the content.

---

I think the linter is currently set to `whatever` regarding this.

Reply:
I didn't mean to make this change, forget it was here. This layout looks like a typo, but I understand it now.